packages/react-native-reanimated/android/src/main/cpp/reanimated/android/NativeProxy.cpp

```diff
@@ -270,6 +275,11 @@ void NativeProxy::registerEventHandler() {
 }
 
 void NativeProxy::maybeFlushUIUpdatesQueue() {
+  // Module might be already destroyed.
+  if (!javaPart_) {
+    return;
+  }
```

Comment:
I'm not a fan of checking this every time, but it sure is the best way if we don't want any refactors here. However, I'd much prefer something among the lines for transparency:

```suggestion
  if (invalidated) {
    return;
  }
```

What do you think?